starport/cmd/network_chain_list.go

```diff
@@ -0,0 +1,89 @@
+package starportcmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
+	"github.com/tendermint/starport/starport/pkg/entrywriter"
+)
+
+var LaunchSummaryHeader = []string{"launch ID", "chain ID", "source", "campaign ID"}
+
+// LaunchSummary holds summarized information about a chain launch
+type LaunchSummary struct {
+	LaunchID   string
+	ChainID    string
```

Comment:
May be it will be more clear
```suggestion
	GenesisChainID    string
```

Reply:
To give some context on why it is called `GenesisChainID`: we used to call it `chainID` because it was at the same time the Cosmos-SDK chain ID (a `string` that uniquely identify your network and act like a nonce like Ethereum Network ID) and the identifier of the chain launch on `spn`
But now we separated both to avoid chain ID squatting. So we name it `GenesisChainID` on `spn` to avoid confusion and make more clear that this is not the identifier on `spn`.

We decided to let as `ChainID` on the CLI side since the confusion is more a problem in the `spn` codebase side.
WDYT?